docs/source/_static/css/custom.css

```diff
@@ -1,94 +1,8 @@
 @import url("theme.css");
 
-section {
```

Comment:
Not super sure if these rules are used in some subpage, have you double-checked with the git history when they were introduced?

Reply:
Will double check that 😄